packages/grafana-ui/src/components/Segment/SegmentAsync.story.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import { SegmentAsync, OptionType } from './';
+import { UseState } from '../../utils/storybook/UseState';
+
+const SegmentStories = storiesOf('UI/Segment/SegmentAsync', module);
+
+const AddButton = (
```

Comment:
This feels like something that we should provide as complete component (that wrapps the other components).

SelectAsyncAdd , SelectSyncAddd ?

Reply:
hmm..maybe. isn't it cleaner just to just export AddButton from grafana UI and pass it on as a component to Segment?